versions/1.1.0-dev.md

```diff
@@ -265,6 +268,200 @@ actions:
 
 This approach allows inversion of control as to where the Overlay updates apply to the target document itself.
 
+#### Copy example
+
+Copy actions behave similarly to `update` actions but source the node to from the document being transformed. Copy `actions` MAY be used in sequence with `update` or `remove` actions to perform more advanced transformations like moving or renaming nodes.
+
+##### Simple copy
+
+This example shows how to copy all operations from the `items` path item to the `some-items` path item.
+
+###### Source description
+
+```yaml
+openapi: 3.1.0
+info:
+  title: API with a paged collection
```

Comment:
We use this title on all the example OpenAPI snippets, but I'm not sure what it's trying to describe. Maybe just "Example API" would make more sense here?

Reply:
```suggestion
  title: Example API
```